kura/distrib/pom.xml

```diff
@@ -2289,25 +2289,40 @@
 											file="${project.build.directory}/plugins/org.eclipse.kura.api_${org.eclipse.kura.api.version}.jar"
 											todir="${project.build.directory}/staging/target-definition/equinox_3.11.1/repository/plugins" />
 										<copy
+											file="${project.build.directory}/plugins/org.eclipse.kura.camel_${org.eclipse.kura.camel.version}.jar"
+											todir="${project.build.directory}/staging/target-definition/equinox_3.11.1/repository/plugins" />
+                                        <copy
```

Comment:
No problem at all. We can have it fixed later on. Anyways it's pretty minor one. 👍